src/native/components/Lotes/LoteHeader.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View } from 'react-native';
+import { Text } from 'native-base';
+
+import Spacer from '../Spacer';
+
+const LoteHeader = ({
+  obrasLength,
+  price,
+  wishlist,
+}) => {
+
+  return (
+    <View>
+      <Spacer />
+      <View style={{
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'space-between'
+      }}>
+        <View style={{
+          display: 'flex',
+          flexDirection: 'row',
+        }}>
+          <Text>{ obrasLength }</Text>
```

Comment:
why are these in separate `<Text>` tags?

Reply:
for spacing i guess. was thinking we would add padding later instead of just using whitespace